pombola/south_africa/templates/menu_entries.html

```diff
@@ -121,12 +128,28 @@
       </div>
       <div class="dropdown-col is--1">
         <p class="dropdown-heading">Activities & Performance</p>
-        <a href="https://pmg.org.za/question_replies/" class="dropdown-link w-inline-block">
-          <p>Questions to ministers</p>
-        </a>
-        <a href="https://pmg.org.za/committees/" class="dropdown-link w-inline-block">
-          <p>Parliamentary committees</p>
-        </a>
+        <div class="dropdown-link__wrapper">
+          <a href="https://pmg.org.za/question_replies/" class="dropdown-link w-inline-block">
+            <p>Questions to ministers</p>
+          </a>
+          <a href="#" class="dropdown-link__info w-inline-block">
+            <div class="svg-icon w-embed svg-small">
+              <img src="{% static 'images/navbar/external-link-alt-solid.svg' %}" loading="lazy" alt=""
+                class="nav__logo-img">
+            </div>
+          </a>
+        </div>
+        <div class="dropdown-link__wrapper">
+          <a href="https://pmg.org.za/committees/" class="dropdown-link w-inline-block">
```

Comment:
Why do we hard code some links with the domain and path and others with just the path? Prefer just path if links to other pages on the same site.

Reply:
this is to a different domain.